frappe/email/queue.py

```diff
@@ -24,7 +24,7 @@ def send(recipients=None, sender=None, subject=None, message=None, text_content=
 		attachments=None, reply_to=None, cc=None, bcc=None, message_id=None, in_reply_to=None, send_after=None,
 		expose_recipients=None, send_priority=1, communication=None, now=False, read_receipt=None,
 		queue_separately=False, is_notification=False, add_unsubscribe_link=1, inline_images=None,
-		header=None, print_letterhead=False):
+		header=None, print_letterhead=False, is_newsletter=False, full_width=False, add_web_link=False):
```

Comment:
Instead of adding a special case for Newsletter, a better way would be to make the width customizable.

This can also be done without making any changes to Email code, but handling it in a separate template specific for Newsletter.